camel/societies/role_playing.py

```diff
@@ -304,7 +308,7 @@ def init_agents(
 
         self.assistant_agent = ChatAgent(
             init_assistant_sys_msg,
-            model_config=assistant_config,
+            model_config=assistant_model_config,
```

Comment:
We might want to remove all these related to `configs` and `assistant_functions` https://github.com/camel-ai/camel/pull/249 since they can be passed from `*_agent_kwargs`.